frontend/src/common/unixToFormattedTimeNoSeconds.ts

```diff
@@ -0,0 +1,10 @@
+export const unixToFormattedTimeNoSeconds = (unixDate?: number) => {
+  if (!unixDate) {
+    return '-'
+  }
+  const date = new Date(unixDate * 1000)
+  const hours = '0' + date.getHours()
```

Comment:
doesn't really seem obvious to me that `hours` is a string, maybe can consider string formatting instead of concatenating number to string?